app/assets/stylesheets/home.css.scss

```diff
@@ -0,0 +1,11 @@
+.home-section {
+  clear:both;
+
+  img { float: left; margin-right:1em; }
+  h4 { padding-top:2em; }
+}
+.baseline_1 img, .scaling img { margin-top: 2em; }
```

Comment:
Selector `baseline_1` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line